### PR TITLE
feat: add support for search term tracking

### PR DESCRIPTION
**Pull request by mzanoni**

### CODE REVIEW

File: lib/src/tracker.ts [hunk 1]
```diff
@@ -97,4 +97,16 @@ export class Tracker extends RelewiseClient {
             }
         });
     }
+
+    public async trackSearchTerm({ term, languageName, user }: { term: string, user: User, languageName: string }): Promise<void | undefined> {
```
Comment (mzanoni) [inline lib/src/tracker.ts:101]: Should this maybe just be `language` ?

Comment (RAORelewise) [reply lib/src/tracker.ts:101]: Imo, yes, to keep it consistent with the .net client

**Review by RAORelewise — APPROVED** [review]
Good find 👍

File: lib/src/tracker.ts (hunk 1 above)
Comment (brianhdk) [reply lib/src/tracker.ts:101]: 👍

**Review by brianhdk — APPROVED** [review]

**Pull request merged**